login.css

```diff
@@ -0,0 +1,86 @@
+@import url(main.css);
+.Img-login
+{
+    width: 100%;
+}
+.row
+{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    justify-content: space-around;
+}
+.col-2
+{
+    flex-basis: 50%;
+    min-width: 18rem;
+}
+.col-2 img{
+    max-width: 100%;
+    padding: 3.5rem;
+}
+.login-page
+{
+    padding: 3.1rem 0;
+    background: radial-gradient(var(--white),var(--black));
+}
+.form-container
+{
+    background: var(--white);
+    width: 20rem;
+    height: 22rem;
+    position: relative;
+    text-align: center;
+    padding: 2rem 0;
+    margin: auto;
+    box-shadow: 0 0 20px 0px rgb(0 0 0 / 10%);
+    border-radius: 10px;
+}
+.form-container span
+{
+    font-weight: bold;
+    padding: 0 10px;
+    color: var(--black);
+    cursor: pointer;
+    width: 6.2rem;
+    display: inline-block;
+    font-size: 2rem;
+}
+.btn{
+    display: inline-block;
+    background: var(--black);
+    color:var(--white);
+    padding: 0.5rem 2.2rem;
```

Comment:
use pixel for less than 20px values

Reply:
i have followed the convention of using pixels for less than 10 in the project